web/src/stream_popover.ts

```diff
@@ -502,6 +507,83 @@ export async function build_move_topic_to_stream_popover(
             is_disabled;
     }
 
+    // Warn if any of current topic participants are NOT subscribed
+    // to the destination stream.
+    function warn_unsubscribed_participants(destination_stream_id: number): void {
+        $("#move_topic_modal .unsubscribed-participants-warning").remove();
+
+        const participants = get_conversation_participants_callback()();
```

Comment:
`get_conversation_participants_callback` seems to only support the case that we're looking at the conversation being moved. Given you can enter this modal from the sidebar or recent/inbox views, I think we can't use that function.

Reply:
`get_conversation_participants_callback` was wrong to begin with in the first place, because it gets the particpants of the currently displayed topic, and a user maybe opens a modal for a topic while viewing the conversation for another.

Reply:
Only `recent_senders.get_topic_recent_senders` and `message_util.get_messages_in_topic` are worth exploring